safe_cli/prompt_parser.py

```diff
@@ -527,4 +531,11 @@ def remove_delegate(args):
     parser_remove_delegate.add_argument("address", type=check_ethereum_address)
     parser_remove_delegate.add_argument("signer", type=check_ethereum_address)
 
+    # Remove not executed proposed transaction
+    parser_remove_proposed_transaction = subparsers.add_parser(
+        "remove_proposed_transaction"
+    )
+    parser_remove_proposed_transaction.set_defaults(func=remove_proposed_transaction)
+    parser_remove_proposed_transaction.add_argument("safe_tx_hash", type=check_hex_str)
```

Comment:
```suggestion
    parser_remove_proposed_transaction.add_argument("safe_tx_hash", type=check_hex_str)
```
Don't we have a parser to check for `keccak` hashes? Maybe we should, because we are expecting an hex string with a defined length

Reply:
Yes, it was done here https://github.com/safe-global/safe-cli/pull/352/commits/3cd2ee94dce6e6ff9c1a96bb35671cef377426ef